src/views/LoginView.vue

```diff
@@ -108,7 +108,8 @@
               </div>
 
               <v-btn
-                class="w-100 my-10 text-h6 h-auto py-2"
+                class="w-100 my-10  h-auto py-2"
```

Comment:
I think we can use vuetifys breakpoints here `text-xs-h4`

https://vuetifyjs.com/en/styles/text-and-typography/

Reply:
Ah yeah so this could replace the dynamic class binding below and keep the classes all together. I think that would be better

Reply:
oh i tried this but i forgot it have mobile first approach and i did something like  this,
```
text-h6  text-xs-subtitle-2 
```
 and this wont work but now i get it what was my  mistake. 
 thanks i will implement you suggestion.